sdk/core/core-tracing/lib/utils/global.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Tracer } from "../interfaces/tracer";
+
+const GLOBAL_TRACER_VERSION = 1;
+const GLOBAL_TRACER_SYMBOL = Symbol.for("@azure/core-tracing.tracerCache");
+
+export interface TracerCache {
+  version: number;
+  tracer?: Tracer;
+}
+
+let cache: TracerCache;
+
+function loadTracerCache(): void {
+  const existingCache = (global as any)[GLOBAL_TRACER_SYMBOL];
+  if (existingCache) {
+    if (existingCache.version !== GLOBAL_TRACER_VERSION) {
+      throw new Error("Two incompatible versions of @azure/core-tracing have been loaded.");
```

Comment:
Any value in saying which versions they are to help with debugging?

Reply:
Yes! I meant to do this originally but forgot 😅